### DISCUSSION TITLE
Allow users to reset dismissed warnings/tips

### DISCUSSION THREAD

**Discussion by turnstylerj** [craftcms/cms#13211] [Ideas]
![Screenshot 2023-05-16 at 4 24 15 PM](https://github.com/craftcms/cms/assets/42223501/eb1000da-cfb0-49b3-bddf-5d6e64a93184)

"Tip" and "Warning" UI Elements can now be set to allow them to be dismissed by users, but there's currently no way (that I can find, at least) for a user to reset their dismissed notifications if they've accidentally dismissed one, or they'd like to bring it back.

It would be nice if there were a button in user preferences to reset their dismissed tips.

**Comment by rynpsc** [comment]
I think switching the behaviour to allow them to be collapsed/expanded rather dismissed entirely would be a good solution rather than hide them entirely in the first place.

The current solution is bound to create confusion where you tell a user to refer to the tip only for them to have dismissed it.

**Reply by thupsi** [reply]
Collapsing them would retain some visual noise in the field layout. Perhaps there could be a discreet button in the entry sidebar to restore tips for the current entry type?

**Reply by RyanRoberts** [reply]
I would appreciate the option too, something like https://media.yep.digital/GrGs8FmW or as thupsi suggests, a button to return them. 

Explaining to a client how they should clear local storage is less than fun.

**Comment by brandonkelly** [comment]
> The current solution is bound to create confusion where you tell a user to refer to the tip only for them to have dismissed it.

You have to opt into a tip being dismissable, so this doesn’t seem like a real concern.
